client/state/login/reducer.js

```diff
@@ -229,15 +228,13 @@ export const twoFactorAuthPushPoll = createReducer(
 );
 
 export const socialAccount = createReducer(
-	{ isCreating: false, createError: null },
+	{ createError: null },
 	{
-		[ SOCIAL_CREATE_ACCOUNT_REQUEST ]: () => ( { isCreating: true } ),
+		[ SOCIAL_CREATE_ACCOUNT_REQUEST ]: () => ( {} ),
```

Comment:
Is `createError: null` missing here? What was the original intention? Does `createError: null` have any benefit over removing the property? Should we improve the initial state?

Reply:
Do we still need to listen to `SOCIAL_CREATE_ACCOUNT_REQUEST` at all?

Reply:
🤷‍♂️ It impacts other properties, this might be clearing old state, for example with `bearerToken` included. Trying to stay focused, although more work here could be beneficial 👍